src/node_report_utils.cc

```diff
@@ -82,6 +83,40 @@ static void ReportEndpoints(uv_handle_t* h, JSONWriter* writer) {
   ReportEndpoint(h, rc == 0 ? addr : nullptr, "remoteEndpoint", writer);
 }
 
+// Utility function to format libuv pipe information.
+static void ReportPipeEndpoints(uv_handle_t* h, JSONWriter* writer) {
+  uv_any_handle* handle = reinterpret_cast<uv_any_handle*>(h);
+  MaybeStackBuffer<char> buffer;
+  size_t buffer_size = buffer.capacity();
+  int rc = -1;
+
+  rc = uv_pipe_getsockname(&handle->pipe, buffer.out(), &buffer_size);
+  if (rc == UV_ENOBUFS) {
+    // Buffer is not large enough, reallocate to the updated buffer_size
+    // and fetch the value again.
+    buffer.AllocateSufficientStorage(buffer_size);
+    rc = uv_pipe_getsockname(&handle->pipe, buffer.out(), &buffer_size);
+  }
+  if (rc == 0 && buffer_size != 0) {
+    writer->json_keyvalue("localEndpointName", buffer.out());
```

Comment:
I wonder if it would be better to stick with the UDP/TCP naming here of just `"*Endpoint"` for consistency?

Reply:
My thought on this is that "localEndpoint"s in tcp/udp are objects with host and port fields, but in pipe case, the field is a string indicating the socket path. However, as the return value of `net_server.address()` is a union type of object and string too, aligning this part may make sense.

Updated. :)